integration_tests/serverless-forwarder.yml

```diff
@@ -87,3 +89,66 @@ functions:
   providedHello:
     handler: provided_handler.hello
     runtime: provided
+
+resources:
+  Resources:
+    stepFunction2LogGroup:
+      Type: AWS::Logs::LogGroup
+      Properties:
+        LogGroupName: /aws/vendedlogs/states/stepFunctionLoggingConfigDynamicArn-Logs-dev
+    stepFunction3LogGroup:
+      Type: AWS::Logs::LogGroup
+      Properties:
+        LogGroupName: /aws/vendedlogs/states/stepFunctionNoLoggingConfigStaticArn-Logs-dev
+
+stepFunctions:
+  stateMachines:
+    stepfunction1:
+      name: stepFunctionNoLoggingConfig
+      definition:
+        StartAt: State1
+        States:
+          State1:
+            Type: Task
+            Parameters:
+              Payload.$: "States.JsonMerge($$, $, false)"
+              FunctionName:
+                Fn::GetAtt: [JavascriptHello18-x, Arn]
+            Resource: arn:aws:states:::lambda:invoke
+            End: true
+    stepfunction2:
+      name: stepFunctionLoggingConfigDynamicArn
+      loggingConfig:
+        level: ALL
+        includeExecutionData: true
```

Comment:
Maybe setting 
```
level: OFF
includeExecutionData: false
```
to verify that our plugin does the configuration correctly.

Reply:
@kimi-p I covered this case in the unit test below. Should I update the integration test to also cover this case?

https://github.com/DataDog/serverless-plugin-datadog/blob/duncan-harvey/step-function-log-groups-serverless-plugin/src/index.spec.ts#L1776-L1830

Reply:
I think unit test should be sufficient. Thank you!